module/VuFind/src/VuFind/Controller/HoldsTrait.php

```diff
@@ -282,4 +283,57 @@ public function holdAction()
         $view->setTemplate('record/hold');
         return $view;
     }
+
+    /**
+     * Email the successful request to staff as a notification.
+     *
+     * @param array $holdDetails Details about the successful hold request
+     *
+     * @return void
+     */
+    protected function emailRequestPlaced($holdDetails)
+    {
+        $config = $this->getConfig();
+        $to = $this->getEmailRecipient($holdDetails);
+        $from = $config->Catalog->holds_email_from ?? null;
+        if (!$to || !$from) {
+            return;
+        }
+
+        $renderer = $this->getViewRenderer();
+        $message = $renderer->render(
+            'Email/request-placed.phtml',
+            ['hold_details' => $holdDetails]
+        );
+
+        $subject = $this->translate('request_email_subject', [
+            '%%id%%' => $holdDetails['id'] ?? '',
+            '%%barcode%%' => $holdDetails['barcode'] ?? '',
+        ]);
+        try {
+            $mailer = $this->getService(\VuFind\Mailer\Mailer::class);
+            $mailer->send(
+                $to,
+                $from,
+                $subject,
+                $message
+            );
+        } catch (\VuFind\Exception\Mail $e) {
+            return;
```

Comment:
I don't want to throw an exception, after all the hold succeeded.  Might be helpful to log the error but I wasn't sure if it was worth using LoggerAwareTrait for the one message.

Reply:
You could also use `error_log` as a last resort.